data/themes/darktable.css

```diff
@@ -562,6 +562,8 @@ overshoot.right
 {
   color: @plugin_label_color;
   border: 0;
+  padding: 0.06em;
+  margin: 0 2px 0 1px;
```

Comment:
Not needed. Just remove those lines for me. I've made many tests and I see that I can move the icon, reduce it but not increase the size. And that's the issue: the always icon maximum size. Probably something on Gtk or Cairo code?